assignments/number-functions/code.js

```diff
@@ -0,0 +1,30 @@
+const add = (a,b) => {
+  return a+b;
```

Comment:
Just as a matter of style (which I'll talk about more in class) you should give operators some room to breath by adding spaces around them (except `!` which is run right against it's operand.) So:

```suggestion
  return a + b;
```